File: src/tests/fuzz_corpus/seed.ts
import { Request, Response } from 'express';
import * as crypto from 'crypto';

export interface Config {
  host: string;
  port: number;
}

export type Handler = (req: Request, res: Response) => void;

export function createServer(config: Config): void {
  console.log(`Starting on ${config.host}:${config.port}`);
}

export class Router {
  private routes: Map<string, Handler> = new Map();
  add(path: string, handler: Handler) { this.routes.set(path, handler); }
}

export enum Status { OK, ERROR, PENDING }

File: src/tests/fuzz_corpus/seed.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;

namespace App.Services
{
    public interface IConfigService
    {
        Config Load();
    }

    public class Config
    {
        public string Host { get; set; }
        public int Port { get; set; }
    }

    public class ConfigService : IConfigService
    {
        public Config Load()
        {
            return new Config { Host = "localhost", Port = 8080 };
        }
    }

    public enum Status { OK, Error, Pending }
}
